### ISSUE TITLE
Add a copy of the license to the plugin folder

### ISSUE THREAD

**Issue by svetogam** [KoBeWi/Godot-Project-Builds#12]
When I install the plugin using the "Install Project Builder Plugin" button, it doesn't come with a license file. Since I'm making a FOSS project, it's valuable to have license files in third-party folders to document what the separate license is for that folder compared to the rest of the project. I have to manually copy the license to that folder if there isn't one. So putting a license there makes things more convenient. (It's also one of the recommendations given by the [Godot docs](https://docs.godotengine.org/en/stable/community/asset_library/submitting_to_assetlib.html).)

**Comment by KoBeWi** [comment]
So you are making a FOSS project that uses the addon as its component?
I never put duplicate license in my addons and no one asked for that since now. Probably because they are used in private projects.
You could just copy the license yourself. Though I could make an exception if that's a problem.

**Comment by svetogam** [comment]
> So you are making a FOSS project that uses the addon as its component?

Yes. That is, I plan to use the addon after the next version is released.

Mostly this is a convenience for others since I already copied the license there. Though depending on how the plugin is updated I might have to copy it again with every update, which would be inconvenient.

The case for doing this for Project Builder is a little better than doing this for other addons. When I click the "Install Project Builder Plugin" button, it creates the addon folder without me seeing it, so I didn't notice immediately that there was no license file there. When installing from the Godot Asset Library it shows the files being installed, so I can more easily notice the lack of a license file in the addon folder. So it's likely that if other people make FOSS projects using this they will forget to copy the license file.
